src/prefect/client/orchestration.py

```diff
@@ -2694,6 +2696,74 @@ async def release_concurrency_slots(
             },
         )
 
+    async def create_global_concurrency_limit(
+        self, concurrency_limit: GlobalConcurrencyLimitCreate
+    ) -> UUID:
+        response = await self._client.post(
+            "/v2/concurrency_limits/",
+            json=concurrency_limit.dict(json_compatible=True, exclude_unset=True),
+        )
+        return UUID(response.json()["id"])
+
+    async def update_global_concurrency_limit(
+        self, concurrency_limit: GlobalConcurrencyLimitUpdate
+    ) -> httpx.Response:
+        concurrency_limit_dict = concurrency_limit.dict(
+            json_compatible=True, exclude_unset=True
+        )
+        current_name = concurrency_limit_dict.get("name")
+        concurrency_limit_dict["name"] = concurrency_limit_dict.get(
+            "rename_to", current_name
+        )
+        if "rename_to" in concurrency_limit_dict.keys():
+            concurrency_limit_dict.pop("rename_to", None)
```

Comment:
Making the name/limit in the update action optional should allow you to remove these checks.

Reply:
I don't know if I don't see something here:
rename_to is no field from the API that can be filled, hence cannot be sent in the json.
Your api supports renaming but when adapting to your suggesting the client doesn't support it.
Example:
There is a limit called "limit". I can do a PATCH-request to <your-url>/api/limit providing this json: {"name": "new-limit"} to rename it. To do this, we need at least two fields containing a name in the client function, hence the variables "name" and "rename_to". "name" always affects the URL to send the request to BUT ALSO the name it shall be renamed to. Hence, in the "name" field in the payload conditionally has to be replaced with "rename_to".

Reply:
Ahh, good point! That's a creative solution to the problem, nice! 

The _alternate_ route here is that we could require the `id` of the concurrency limit and since the API supports either a name or an id you could just do what I was suggesting. But I think I like your approach better, it keeps the user from having to make a second api call to get the id to do the update.

Reply:
That's true, using an ID would result in distinguishable variable names. But as you said, IDs have some kind of overhead to them and names are more intuitive.
Thanks for your constructive and helpful feedback anyways. I'll push another quick update :)